Gemfile.lock

```diff
@@ -16,7 +16,7 @@ GEM
     docile (1.3.2)
     ethon (0.12.0)
       ffi (>= 1.3.0)
-    ffi (1.13.1)
+    ffi (1.14.2)
```

Comment:
Why did this happen? 😊

Reply:
To remove the Teamtailor-version I earlier updated from gemlock, I ran "bundle". Ffi 1.14.2 is the newest version of the gem I guess :)

Reply:
Do you want me to revert this?

Reply:
Yes please! 😊

Reply:
Done! :)